cmd/kubeadm/app/util/version.go

```diff
@@ -22,6 +22,7 @@ import (
 	"net/http"
 	"regexp"
 	"strings"
+	"testing"
```

Comment:
The importing of `testing` package for a release binary seems to be prohibited.
Maybe we can only control it by adding additional parameter for `KubernetesReleaseVersion` func.

Reply:
> The importing of testing package for a release binary seems to be prohibited.

i think it's an odd requirement, because the compiler will only consider required symbols.
https://github.com/kubernetes/kubernetes/pull/121735
https://github.com/kubernetes/kubernetes/pull/122963

> Maybe we can only control it by adding additional parameter for KubernetesReleaseVersion func.

it's a bigger refactor.

Reply:
updated.